README.md

```diff
@@ -6,6 +6,127 @@ interface. The runtime for this application is configured using a `config.json`
 `config.example.json`, however it only has the `GoogleGroups` destination in it so other supported destinations are 
 documented below. 
 
+# Config
+
+## Email Alerts
+
+Event Log events with a level of LOG_ALERT or LOG_EMERG will result in an email
+alert sent via AWS SES. Note that the LOG_EMERG level is 0, which is the Go
+zero-value. Any new event log created without a Level assigned will default to
+LOG_EMERG and could result in email alerts being sent.
+
+The following is an example configuration:
+
+```json
+{
+  "Alert": {
+    "AWSRegion": "us-east-1",
+    "CharSet": "UTF-8",
+    "ReturnToAddr": "no-reply@example.org",
+    "SubjectText": "personnel-sync alert",
+    "RecipientEmails": [
+      "admin@example.org"
+    ],
+    "AWSAccessKeyID": "ABCD1234",
+    "AWSSecretAccessKey": "abcd1234!@#$"
+  }
+}
+```
+
+Alternatively, AWS credentials can be supplied by the Serverless framework by adding
+the following configuration to `serverless.yml`:
+
+```
+provider:
+  iamRoleStatements:
+    - Effect: 'Allow'
+      Action:
+        - 'ses:SendEmail'
+      Resource: "*"
+```
+
+Both authentication mechanisms are provided in the `lambda-example` directory,
+but only one is needed.
+
+## Pagination
+
+### `RestAPI`
+
+The RestAPI adapter supports pagination as both a source and as a destination.
+
+#### Properties:
+- Scheme -- if specified, must be "pages" for page based or "items" for item based
+- FirstIndex -- index of first item/page to fetch, default is 1
+- NumberKey -- query string key for the item index or page number
+- PageLimit -- index of last page to request, default is 1000
+- PageSize -- number of records to return in a page, default is 100
+- PageSizeKey -- number of items per page, default is 100
+
+#### Example config
+
+Following is an example configuration for Pagination. Unrelated parameters have 
+been omitted for simplicity.
+
+```json
+{
+  "Source": {
+    "Type": "RestAPI",
+    "ExtraJSON": {
+      "Pagination": {
+        "Scheme": "pages",
+        "NumberKey": "page",
+        "PageSizeKey": "page-size",
+        "FirstPage": "1",
```

Comment:
Sorry. I hadn't finished reviewing the other PR and got distracted.
This should be `FirstIndex`, right?

Reply:
Correct, thanks.